accounts/validators.py

```diff
@@ -0,0 +1,15 @@
+from django.core.exceptions import ValidationError
+from django.utils.translation import ugettext as _
+import re
+
+
+class ComplexPasswordValidator(object):
```

Comment:
please add tests for this new object. The test should (a) test the individual methods, and (b) configure this validator in settings (you can override settings configuration on a per test method/test case basis) and make sure it performs properly when active as a validator